gimme

```diff
@@ -286,6 +287,17 @@ _wipe_version() {
 	fi
 }
 
+_list_versions() {
+	if [ ! -d "${GIMME_VERSION_PREFIX}" ] ; then
+		return 0
+	fi
+
+	for d in "${GIMME_VERSION_PREFIX}/go"*".${GIMME_OS}."* ; do
+		local lessgo="${d##*/go}"
+		echo "${lessgo%%.${GIMME_OS}.*}"
+	done
```

Comment:
As a rule, I avoid scripting the output of `ls` like the plague that it is, especially since Bash by itself or `find` if absolutely desperate can usually fit the bill. :+1: